src/clj/kixi/hecuba/api/measurements.clj

```diff
@@ -34,6 +34,12 @@
 
 (def ^:private uploads-status-resource-path (p/resource-path-string :uploads-status-resource))
 
+(defn get-measurements-keys [sensor-keys session]
+  (let [sensor (sensors/get-by-type sensor-keys session)]
+    (if-let [alias-sensor (:alias-sensor sensor)]
```

Comment:
You are essentially return a map of keys. The id's are uuid type keys, not integers.